lighthouse-core/lib/rect-helpers.js

```diff
@@ -68,6 +63,7 @@ function filterOutRectsContainedByOthers(rects) {
 /**
  * @param {LH.Artifacts.Rect} rect
  */
+/* istanbul ignore next */
```

Comment:
doesn't this get coverage info from use in `tappable-rects.js`, so don't need to ignore?

Reply:
I think nyc renames the functions when doing the code coverage test. So we disable it on all functions that are injected into the gatherer with `.toString()`.

Reply:
> I think nyc renames the functions when doing the code coverage test

ah. That's really annoying :)

we really should switch to native node coverage if we can...